lib/json.js

```diff
@@ -0,0 +1,84 @@
+var indexOf = [].indexOf || function(item) { for (var i = 0, l = this.length; i < l; i++) { if (i in this && this[i] === item) return i; } return -1; };
+
+module.exports = {
+  parse: function(graph, options) {
```

Comment:
The function should take a JS object of the graph, not a string with JSON encoded data.

I think it is misleading to call this a parser. It serializes to the .FBP format. It is the inverse function of `fbp.parse`